File: SampleApp/Objective-C/SampleApp/Banner/BannerTableViewCell.m
//
//  BannerTableViewCell.m
//  SampleApp
//
//  Fluct SDK
//  Copyright (c) 2020 fluct, Inc. All rights reserved.

#import "BannerTableViewCell.h"
#import <FluctSDK/FluctSDK.h>

@interface BannerTableViewCell ()
@property (nonatomic, nullable) FSSAdView *adView;
@end

@implementation BannerTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];

    FSSAdView *adView = [[FSSAdView alloc] initWithGroupId:@"1000055927" unitId:@"1000084701" adSize:FSSAdSize320x50];
    [self.contentView addSubview:adView];

    [adView loadAd];
    self.adView = adView;
}

- (void)layoutSubviews {
    [super layoutSubviews];
    self.adView.center = self.contentView.center;
}

@end

File: SampleApp/Swift/SDK/SampleApp/Banner/BannerTableViewCell.swift
//
//  BannerTableViewCell.swift
//  SampleApp
//
//  Fluct SDK
//  Copyright © 2020年 fluct, Inc. All rights reserved.

import UIKit
import FluctSDK

class BannerTableViewCell: UITableViewCell {

    private var adView: FSSAdView?

    override func awakeFromNib() {
        super.awakeFromNib()

        let adView = FSSAdView(groupId: "1000055927", unitId: "1000084701", adSize: FSSAdSize320x50)
        self.contentView.addSubview(adView)
        adView.loadAd()
        self.adView = adView
    }

    override func layoutSubviews() {
        super.layoutSubviews()
        adView?.center = self.contentView.center
    }

}
